app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_persons = [(Person(_["name"], _["age"])) for _ in people]
+
+    for _ in people:
```

Comment:
it is bad practice to use "_" variable for loop if u will use it later, use "_" only if there is no need to use this variable